easybuild/easyconfigs/s/SciPy-bundle/SciPy-bundle-2023.11-iimkl-2023b.eb

```diff
@@ -0,0 +1,130 @@
+easyblock = 'PythonBundle'
+
+name = 'SciPy-bundle'
+version = '2023.11'
+
+homepage = 'https://python.org/'
+description = 'Bundle of Python packages for scientific software'
+
+toolchain = {'name': 'iimkl', 'version': '2023b'}
+toolchainopts = {
+    'pic': True,
+    'lowopt': True,
+    'strict': True,
+    'oneapi': True,
+    'optarch': False,  # prevents -xHost
```

Comment:
@Louwrensth Can you clarify why this is required?

None of the existing `SciPy-bundle` easyconfigs have this (which doesn't mean it may not be a necessary evil here)

Reply:
> Can you clarify why this is required?

I don't know if it is, I got the EB like this from my old stash -- I was likely trying this without intending to keep it.